homeassistant/components/fritzbox/binary_sensor.py

```diff
@@ -32,6 +29,22 @@ def __init__(self, device, fritz):
         self._device = device
         self._fritz = fritz
 
+    @property
+    def device_info(self):
```

Comment:
Same comment as above about consolidation. In **another PR** we could create a common fritz entity base class that holds the common attributes.